### ISSUE TITLE
Make it possible to configure DENO_DIR in WS settings

### ISSUE THREAD

**Issue by Tzahile** [denoland/vscode_deno#287]
### Is your feature request related to a problem? Please describe.
I made a **Workspace** for a project of mine written with Deno.
For the sake of convenience I set `DENO_DIR` to be the project directory.
Every time I enter my workspace, I need to configure `DENO_DIR`. Until then, many things don't work and the `Deno` extension misbehaves.

### Describe the solution you'd like

Would be great if `DENO_DIR` setting can be configures in the workspace settings, just like the `unstable` option

**Comment by kitsonk** [comment]
Yeah. This is a good idea. We will look to add it.

**Comment by tehbeard** [comment]
Documenting this here, as it was a workaround I came up with before #477 is implemented and merged.

I tried using a batch file to set `DENO_DIR` and call the batch file instead.

```
@Echo off
SETLOCAL
set DENO_DIR=E:\deno\.deno
E:\deno\deno.exe %*
ENDLOCAL
```

While this did seem to work, vscode/the plugin wouldn't kill the process when vscode exited, which left it hanging in memory, and also locked workspace folders as "in-use" so they couldn't be deleted/renamed in explorer.
